browser-test/src/admin_program_creation.test.ts

```diff
@@ -66,9 +67,7 @@ describe('program creation', () => {
       '.cf-program-question:has-text("apc-name") >> .cf-remove-question-button',
     )
     await page.click('button:text("apc-enumerator")')
-    expect(await page.innerText('id=question-bank-questions')).toBe(
-      'Add Question',
-    )
+    expect(await page.innerText('id=question-bank-questions')).toBe('')
```

Comment:
Note to the reviewer: This change and the other is due to the DOM restructuring in QuestionBank. We were applying this ID to the entire question bank container rather than something containing the list of questions, thus causing us to over-specify in these sort of expectations.

We now have a screenshot test if we truly care about asserting that the 'Add Question' text is there.